src/main/java/core/basesyntax/service/impl/WriterToCsvImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterToCsv;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterToCsvImpl implements WriterToCsv {
+    @Override
+    public void write(String data, String toFileName) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            bufferedWriter.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write to file" + toFileName, e);
```

Comment:
The exception message in the RuntimeException should include a space before the file name for better readability: "Can't write to file " + toFileName.